examples/dbpedia/dbpedia_study_config.yaml

```diff
@@ -0,0 +1,30 @@
+input_data_type: "json" # currently only json but considering alternatives
+raw_data_path: "data/dbpedia_250k.json" # path to data files for easy read
```

Comment:
These dataset should be uploaded to S3 if we want them as part of the public examples